spec/controllers/hyrax/admin/admin_sets_controller_spec.rb

```diff
@@ -147,6 +148,12 @@
       let(:admin_set) { create(:admin_set, edit_users: [user]) }
 
       context "with empty admin set" do
+        before do
+          allow_any_instance_of(ActionController::TestRequest).to receive(:referer) do
```

Comment:
using `allow_any_instance_of` is discouraged. Can you do `allow(controller).to receive(:request).and_return(double(referer: '/path'))` ?

Reply:
This one does not work. Have tried many variations, but I'm not having any luck.

Reply:
Okay, lets try: `request.env['HTTP_REFERER'] = '/path'`

Reply:
I found a solution:
controller.request.set_header("HTTP_REFERER", "/my/collections")